src/sentry/static/sentry/app/utils/withIssueTags.tsx

```diff
@@ -72,25 +81,26 @@ const withIssueTags = <P extends InjectedTagsProps>(
     },
 
     setAssigned() {
-      if (this.state.users && this.state.tags.assigned) {
-        const {tags, users} = this.state;
-        const usernames: string[] = users.map(getUsername);
-        usernames.unshift('me');
-
-        this.setState({
-          tags: {
-            ...tags,
-            assigned: {
-              ...tags.assigned,
-              values: usernames,
-            },
-            bookmarks: {
-              ...tags.bookmarks,
-              values: usernames,
-            },
+      const {tags, users, teams} = this.state;
+      const usernames: string[] = users.map(getUsername);
+      const teamnames: string[] = teams
+        .filter(team => team.isMember)
+        .map(team => `#${team.name}`);
+      const allAssigned = usernames.concat(teamnames);
+      allAssigned.unshift('me') && usernames.unshift('me');
```

Comment:
nit: this is a little strange to one line

Reply:
@scttcper ah ok cool, would you recommend something like this:

```
const allAssigned = [usernames.concat(teamnames), usernames];
allAssigned.map(assigned => assigned.unshift('me'));
```

Reply:
the same unshift you have on two lines is fine. It always does both right?

Reply:
ok cool cool, and yeah it'll always do both